src/components/SelectionCart/SelectionCart.js

```diff
@@ -52,13 +55,32 @@ class SelectionCart extends Component {
             'reservation': [
             ]
         };
+        let reservedComponents = {
```

Comment:
No, then pushes some components on it

Reply:
Pushing elements into a list inside an object doesn't affect the type of the object itself. The definition of const means that the variable can't be re-assigned or re-declared in the script, but anything inside it **can in fact** be mutated. 

```
// This is wrong, TypeError....
const foo = {x: 1}
foo = {x: 7}
```
```
// This is ok
const foo = {x: 1}
foo.x = 7
```
For what I'm seeing in the code, you're never re-assigning the variable, just adding elements to it's list, which in this case it can be changed to `const`